_posts/2017-06-12-ropensci_text_tools.md

```diff
@@ -0,0 +1,157 @@
+---
+name: ropensci_text_tools
+layout: post_discourse
+title: "New rOpenSci Packages for Text Procesing in R"
+date: 2017-06-12
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- opendata
+- pdf
+---
+
+At the rOpenSci [text workshop](http://textworkshop17.ropensci.org/) in April we discussed many ideas for improving text processing in R. Over the past weeks I was able to work on some of these suggestions and we released a few packages for reading and analyzing text. 
```

Comment:
use this link for text workshop: https://ropensci.org/blog/blog/2017/05/03/textworkshop17